en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -11,7 +72,81 @@ We will only consider merging post-beta breaking API changes if they are necessa
+- silverstripe/cms [![CI](https://github.com/silverstripe/silverstripe-cms/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/silverstripe-cms/actions/workflows/ci.yml)
+- silverstripe/elemental-banner-block [![CI](https://github.com/silverstripe/silverstripe-elemental-bannerblock/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/silverstripe-elemental-bannerblock/actions/workflows/ci.yml)
+- silverstripe/session-manager [![CI](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml)
+- silverstripe/recipe-content-blocks [![CI](https://github.com/silverstripe/recipe-content-blocks/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/recipe-content-blocks/actions/workflows/ci.yml)
+- silverstripe/recipe-core [![CI](https://github.com/silverstripe/recipe-core/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/recipe-core/actions/workflows/ci.yml)
+- silverstripe/staticpublishqueue [![CI](https://github.com/silverstripe/silverstripe-registry/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/silverstripe-registry/actions/workflows/ci.yml)
+
+#### Other issues
+
+The CMS 5 compatible version of `tractorcow/silverstripe-fluent` is temporarily being released under `silverstripe/silverstripe-fluent`.
+
+View the [Silverstripe CMS 5 compatibility](https://github.com/tractorcow-farm/silverstripe-fluent/issues/776) issue on the Fluent repository for progress.
+
+## Upgrading your third party module
+
+One of the primary purposes of the beta testing phase is to give third party maintainers and developers an opportunity to upgrade their modules to be compatible with Silverstripe CMS 5.
+
+### Planning your upgrade
+
+You should decide if you intend to do a new major release of your module to add support for Silverstripe CMS 5.
+
+Most of the key Silverstripe CMS APIs have not changed between CMS 4 and CMS 5. That said, take time to read this changelog to identify changes that might be problematic for your project.
+
+Depending on how complex your module is, it may be completely practical to create a version that works with both Silverstripe CMS 4 and Silverstripe CMS 5. We do not necessarily recommend this approach. It may require you to tag a major release later on if you want to make use of new SilverstripeCMS 5 feature later on. But you may wish to consider this approach to minimise your maintenance burden.
+
+If you decide that you want to maintain distinct Silverstripe CMS 4 and Silverstripe CMS 5 versions of your module, then you should create a new branch from your current development branch.
```

Comment:
This is still a bit unclear what we're actually telling people to do... let's give an example, something like
> If your current CMS 4 compatible version is on the `main` branch, create a new branch off `main` which is named after the current major release of your module - for example, if your current major version is `3`, create a branch called `3` which will continue to be your CMS 4 compatible version. You can then use your `main` branch for the CMS 5 compatible version, and still be able to provide updates to your old version.

That's obviously not quite the right words, that sounds clunky... but I think we need to make it clear what we're saying here if we're offering this advice or it's not obvious _why_ anyone would want to do it (except to people like us, who would already do it).

Reply:
Added a merge up crash course.